### DISCUSSION TITLE
Fuzzy match accented characters

### DISCUSSION THREAD

**Discussion by toby1kenobi** [TarekRaafat/autoComplete.js#421] [General]
Is it possible to match unaccented characters like a and o with accented ones, like á and ó (for example)?

Thanks

**Comment by TarekRaafat** [comment]
Hello @toby1kenobi,

Apologies for my delayed reply.

Yes, this is possible using the `diacritics` config option. Setting `diacritics: true` normalizes both the query and the data values, so searching for "a" will match "á", "à", "â", etc.

```js
const ac = new autoComplete({
  selector: "#autoComplete",
  diacritics: true,
  data: { src: ["café", "naïve", "résumé"] }
});
```

Please try it and let me know how it goes.

Cheers! :)
